controllers/C_Document.class.php

```diff
@@ -660,7 +660,7 @@ function queue_action_process() {
 			
 			//validate that the pod exists
 			$d = new Document($doc_id);
-			$sql = "SELECT pid from patient_data where pubpid = '" . $file['patient_id'] . "'";
```

Comment:
Yes, it is foreign_id.
This would require a conversion script that says:
UPDATE documents d
        INNER JOIN
    patient_data p ON p.pubpid = d.foreign_id 
SET 
    d.foreign_id = p.pid;